client/rpcserver/handlers.go

```diff
@@ -127,9 +128,7 @@ func handleInit(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 	}
 	defer func() {
 		appPass.Clear()
-		if len(seed) > 0 {
-			seed.Clear()
-		}
```

Comment:
Often confused about slice being nil and not really nil. Confirming this will not panic https://go.dev/play/p/DS8Zp9uQFCy

Reply:
OH! I've been misunderstanding when things panic for a while then.